02_flow_compensation/exercises.m

```diff
@@ -154,18 +164,20 @@
 % 2) Whether to map all values <= 0 to 1 (which is zero on the log scale)
 %    We typically use this when data is going to be interpreted on the log scale,
 %    and we aren't planning to filter it ourselves differently later,
-%    but it has the unfortunate side effect of creating a large number of 1s
+%    but it has the unfortunate side effect of creating a large number of
+%    1s. We can also remove negative values before compensation. 
```

Comment:
"... before compensation by adding a gate that filters them out."

Reply:
done